gatsby-config.ts

```diff
@@ -2,14 +2,28 @@ import type { GatsbyConfig } from "gatsby";
 
 const config: GatsbyConfig = {
   siteMetadata: {
-    title: `blazerbots`,
+    title: `Overland BlazerBots FRC Team 3807`,
+    description: "The official website of FRC Team 3807, the Overland BlazerBots",
     siteUrl: `https://www.blazerbots.org`,
   },
   // More easily incorporate content into your pages through automatic TypeScript type generation and better GraphQL IntelliSense.
   // If you use VSCode you can also use the GraphQL plugin
   // Learn more at: https://gatsby.dev/graphql-typegen
   graphqlTypegen: true,
-  plugins: [],
+  plugins: [
+    {
+      resolve: `gatsby-plugin-manifest`,
+      options: {
+        name: "Overland BlazerBots FRC Team 3807",
+        short_name: "BlazerBots",
+        start_url: "/",
+        background_color: "#ff007f",
+        theme_color: "#ff007f",
+        display: "standalone",
+        icon: "src/images/logo.svg", // This path is relative to the root of the site.
```

Comment:
This is reusing the logo we already have defined in `images/` for the favicon (`gatsby-plugin-manifest` takes care of resizing). I posted a question in Teams regarding using alternate versions. If we decide to use a different icon/color scheme all we will need to do is upload that version, and point to that file here.

Reply:
As-is which version from teams is used?

Reply:
This would be option No. 6 (pink fill with transparent background). For the post in Teams I modified the SVG in various combinations to show how it could look (work to display those modifications are not part of this PR).